vimrc

```diff
@@ -359,3 +359,41 @@ endif
 
 " ensure vim always runs from a shell
 set shell=/bin/sh
+
+
+""""""""""""""""""""""
```

Comment:
@ryanhinkel, it's  important that this configuration comes *before* loading `vimrc.local` otherwise, people can't override these settings.